library/core/src/str/mod.rs

```diff
@@ -454,7 +454,7 @@ impl str {
     /// assert!(v.get_mut(0..5).is_some());
     /// // out of bounds
     /// assert!(v.get_mut(..42).is_none());
-    /// assert_eq!(Some("he"), v.get_mut(0..2).map(|v| &*v));
+    /// assert_eq!(Some("they"), v.get_mut(0..2).map(|v| &*v));
```

Comment:
Hello, @jamiepinheiro Your bot is busted. "he" is used here not as a pronoun but as a random two-character string.